uiowa/uiowa
# Hotfix: Added missing views to UIPress node person page and field delimiter to fields

Resolves https://github.com/uiowa/uiowa/issues/6404. 

# how to test

```
ddev blt ds --site=uipress.uiowa.edu
```
Add multiple collaborators for each field in https://uipress.uiowa.ddev.site/node/11836/edit and verify that each field is comma separated. 

<img width="921" alt="Screenshot 2023-05-03 at 9 47 10 AM" src="https://user-images.githubusercontent.com/1036433/235952055-945ff56e-c173-4246-9c6b-08cf3abf0c85.png">

Edit one of the persons added to that book like https://uipress.uiowa.ddev.site/node/211/edit and select all person types:

<img width="780" alt="Screenshot 2023-05-03 at 9 48 10 AM" src="https://user-images.githubusercontent.com/1036433/235952451-6f9a1ec1-7aa1-4c01-bc8c-420d2bb9c536.png">

Verify that Author, Editor, Illustrator, Photographer, Translator views show up. 

<img width="917" alt="Screenshot 2023-05-03 at 9 43 53 AM" src="https://user-images.githubusercontent.com/1036433/235952517-37147351-6796-40aa-b54a-b7d5e6647c73.png">

Also verify that authors in "Related Reads" section are comma separated:

<img width="1495" alt="Screenshot 2023-05-03 at 9 49 38 AM" src="https://user-images.githubusercontent.com/1036433/235952759-8510e71a-4743-40df-9e35-cbc13fe54b7c.png">
